packages/scaffold-ui/src/partials/w3m-social-login-list/index.ts

```diff
@@ -37,12 +48,74 @@ export class W3mSocialLoginList extends LitElement {
 
     return html` <wui-flex flexDirection="column" gap="xs">
       ${this.connector.socials.map(
-        social => html`<wui-list-social name=${social} logo=${social}></wui-list-social>`
+        social =>
+          html`<wui-list-social
+            @click=${() => {
+              this.onSocialClick(social)
+            }}
+            name=${social}
+            logo=${social}
+          ></wui-list-social>`
       )}
     </wui-flex>`
   }
 
   // -- Private ------------------------------------------- //
+  async onSocialClick(socialProvider?: SocialProvider) {
+    if (socialProvider) {
+      AccountController.setSocialProvider(socialProvider, ChainController.state.activeChain)
+
+      EventsController.sendEvent({
+        type: 'track',
+        event: 'SOCIAL_LOGIN_STARTED',
+        properties: { provider: socialProvider }
+      })
+    }
+    if (socialProvider === 'farcaster') {
```

Comment:
Do we have an enum for social options - we'd better to use enums/constants rather than hard coded strings